csrc/ops/arith.h

```diff
@@ -716,4 +734,29 @@ NVF_API TensorView* argsort(
     bool descending = false,
     bool stable = false);
 
+//! TopK operation: find the k largest or smallest elements along a dimension
+//!
+//! Returns the k largest (if largest=true) or smallest (if largest=false)
+//! elements of the input tensor along the given dimension.
+//!
+//! \param v1 Input tensor
+//! \param k Number of elements to return (must be non-negative integer)
+//! \param dim Dimension along which to find top-k elements (default: -1, last
+//! dim)
+//! \param largest If true, return largest elements; if false, return smallest
+//! (default: true)
+//! \param sorted If true, return elements in sorted order (default: false)
+//! \param maybe_symbolic Whether to allow symbolic shapes (default: true)
```

Comment:
This comment doesn't seem to make sense?

Reply:
Never mind, I get it.

Maybe it would be helpful to mention this should be always true except when `k` is guaranteed to be not one.